packages/oui-clipboard/src/clipboard.provider.js

```diff
@@ -0,0 +1,42 @@
+export default class {
+    constructor () {
+        this.translations = {
+            copy_to_clipboard_label: "Copy to Clipboard",
+            copied_label: "Copied"
+        };
+        this.action = {
+            copy: "copy"
+        };
+        this.status = {
+            initial: "initial",
+            success: "success"
+        };
+    }
+
+    /**
+     * Set the translations
+     * @param {Object} translations a map of translations
+     */
+    setTranslations (translations) {
+        this.translations = translations;
+        return this;
+    }
+
+    setStatus (status) {
+        this.status = status;
+        return this;
+    }
+
+    setAction (copy) {

```

Comment:
Same for this one too :)

Reply:
using these functions in spec for test cases